src/picongpu/include/simulation_defines/param/ionizerConfig.param

```diff
@@ -270,6 +277,104 @@ namespace effectiveNuclearCharge
         28.339,
         28.339
     );
+
+    /* Example: gold */
+    PMACC_CONST_VECTOR(float_X, 79, Gold,
+        /* 5d */
+        20.126,
+        20.126,
+        20.126,
+        20.126,
+        20.126,
+        20.126,
+        20.126,
+        20.126,
+        20.126,
+        /* 4f */
+        40.650,
+        40.650,
+        40.650,
+        40.650,
+        40.650,
+        40.650,
+        40.650,
+        40.650,
+        40.650,
+        40.650,
+        40.650,
+        40.650,
+        40.650,
+        40.650,
+        /* 6s */
+        10.938,
+        10.938,
+        /* 5p */
+        25.170,
+        25.170,
+        25.170,
+        25.170,
+        25.170,
+        25.170,
+        25.170,
+        /* 4d */
+        41.528,
+        41.528,
+        41.528,
+        41.528,
+        41.528,
+        41.528,
+        41.528,
+        41.528,
+        41.528,
+        41.528,
+        /* 5s */
+        27.327,
+        27.327,
+        /* 4p */
+        43.547,
+        43.547,
+        43.547,
+        43.547,
+        43.547,
+        43.547,
+        /* 3d */
+        65.508,
+        65.508,
+        65.508,
+        65.508,
+        65.508,
+        65.508,
+        65.508,
+        65.508,
+        65.508,
+        65.508,
+        /* 4s */
+        44.413,
+        44.413,
+        /* 3p */
+        56.703,
+        56.703,
+        56.703,
+        56.703,
+        56.703,
+        56.703,
+        /* 3s */
+        55.763,
+        55.763,
+        /* 2p */
+        74.513,
```

Comment:
I really wonder what the implications are that the effective Z is not strictly ascending. Right now my own personal experimental BSI model is the only thing that uses them but maybe one should treat the whole thing differently. Haven't looked into that.